src/components/account-menu/account-menu.yaml

```diff
@@ -55,6 +55,23 @@ params:
     type: string
     required: false
     description: Flag to mark the Your profile item as active or not.
+- name: businessTaxAccount
+  type: object
+  required: false
+  description: The Business Tax Account item.
+  params:
+  - name: href
+    type: string
+    required: false
+    description: "The url of the business-tax-account page in the personal tax account. Defaults to #."
+  - name: active
+    type: string
+    required: false
+    description: Flag to mark the Business tax account as active or not.
+  - name: visible
```

Comment:
@kyle-bowden the visible argument slipped in

Reply:
Thanks, i will remove it.